### DISCUSSION TITLE
is that possible to use appwrite without docker?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#1466] [Q&A] [answered]
I can create environment for PHP, mairadb, redis etc.. is that possible to run appwrite without docker?

**Comment by TorstenDittmann** [comment, marked as answer]
It is in theory possible to run Appwrite without Docker, but that is not recommended for multiple reasons.

- scaling
- consistency
- maintainability
- upgrading to a new version of Appwrite
- Cloud Functions require Docker anyway

The steps to achieve this would be to imitate our `Dockerfile` and `docker-compose.yml` file.
